libpod/networking_linux.go

```diff
@@ -556,7 +556,7 @@ func (r *Runtime) setupRootlessPortMappingViaRLK(ctr *Container, netnsPath strin
 		if stdoutStr != "" {
 			// err contains full debug log and too verbose, so return stdoutStr
 			logrus.Debug(err)
-			return errors.Errorf("failed to expose ports via rootlessport: %q", stdoutStr)
+			return errors.Errorf(strings.TrimSuffix(stdoutStr, "\n"))
```

Comment:
I'm a little iffy about removing any reference to RootlessPort - we want to leave some breadcrumbs so developers can find where the error came from.

Reply:
Well I found this code by grep'ing through the vendor directory.  Not based on the first part.
Would you be ok with:
return errors.Errorf("rootlessport: " + strings.TrimSuffix(stdoutStr, "\n"))

Reply:
Sure, I'm ok with that. I just want something that I can easily grep for to figure out where an error message is coming rom